testing/scripts/test_helpdesk_pw.py

```
@@ -134,14 +135,23 @@ def _optional(name, default=""):
-    """Run a WP-CLI command inside the Docker container via SSH, return stdout stripped."""
-    docker_cmd = (
-        f"docker exec {WP_CONTAINER} wp {cmd} --path={WP_PATH} --allow-root 2>/dev/null"
-    )
-    result = subprocess.run(
-        ["ssh", SSH_HOST, docker_cmd],
-        capture_output=True, text=True, timeout=15
-    )
+    """Run a WP-CLI command. Uses docker compose exec (WP_MODE=docker) or SSH+docker exec."""
+    if WP_MODE == "docker":
+        repo_root = os.path.normpath(os.path.join(os.path.dirname(__file__), "..", ".."))
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)